src/webgl/interaction.js

```diff
@@ -19,19 +19,30 @@ var p5 = require('../core/core');
  * <code>
  * function setup() {
  *   createCanvas(100, 100, WEBGL);
+ *   normalMaterial();
  * }
- *
  * function draw() {
  *   background(200);
- *   // Orbit control allows the camera to orbit around a target.
  *   orbitControl();
- *   box(30, 50);
+ *   translate(-20, 15, -20);
+ *   torus(20, 5);
+ *   translate(15, 0, 15);
+ *   torus(20, 5);
+ *   translate(15, 0, 15);
+ *   torus(20, 5);
+ *   translate(15, 0, 15);
+ *   torus(20, 5);
+ *   translate(15, 0, 15);
+ *   torus(20, 5);
+ *   translate(15, 0, 15);
+ *   torus(20, 5);
  * }
  * </code>
  * </div>
  *
  * @alt
- * Camera orbits around box when mouse is hold-clicked & then moved.
+ * Camera orbits around a series of floating shapes when mouse is
+ * hold-clicked & then moved.
  */
 //@TODO: implement full orbit controls including
 //pan, zoom, quaternion rotation, etc.
```

Comment:
what's left from this TODO? anything? :)

Reply:
Just added zooming, so unless anyone wants to implement quaternions, I think not.  Removed.